src/examples/color-swatch/ColorSwatchDefault.tsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { ColorSwatch } from '@zendeskgarden/react-colorpickers';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { convertToMatrix } from '@zendeskgarden/container-utilities';
+import { labeledColors } from '../../../react-components/packages/colorpickers/stories/examples/utils';
+
+export const colors = labeledColors(
+  DEFAULT_THEME.palette,
+  ['green', 'red', 'blue', 'yellow'],
+  ['200', '300', '400', '500', '600', '700', '800']
+);
```

Comment:
Need to build this matrix (using `PALETTE` rather than theme) by hand. This construct breaks for CodeSandbox.

Reply:
Looks like the relative `import` path isn't supported. The `labeledColors` function seems a bit too large to include in here. 

I'm thinking the `colorpickers` package could export the `labeledColors` utility. Or the `colors` data could be hardcoded in the example with Prettier ignore. Do either of these sound reasonable?

Reply:
All website examples need to use local constructs. So hard-coding is the right option.